tests/queries/0_stateless/01410_nullable_key_more_tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Tags: no-parallel
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+
+echo ${CLICKHOUSE_URL} > /tmp/xxx
```

Comment:
Probably you forgot to remove this line. If you need a temporary file for test, use `${CLICKHOUSE_TMP}` directory.